LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Franklin Harding, Brendan Burkhart, Caleb Eby et al.
```

Comment:
Nit: fix the year

Should we change this to Pigmice instead of our individual names?

Reply:
Is Pigmice an organization that can actually hold a copyright? We're not a non-profit as far as I can tell, and I don't think we have any other legal status.

Reply:
Hm, I'm not sure, I would like to know more about this